timemachine/md/minimizer.py

```diff
@@ -349,10 +354,36 @@ def val_and_grad_fn_bfgs(x_local_flattened):
     x_local_0 = x0[local_idxs]
     x_local_0_flat = x_local_0.reshape(-1)
 
-    res = scipy.optimize.minimize(val_and_grad_fn_bfgs, x_local_0_flat, method="BFGS", jac=True, options={"disp": True})
+    method = "BFGS"
+
+    if verbose:
+        print("-" * 70)
```

Comment:
Non blocking: in the future we might consider using [rich](https://github.com/Textualize/rich) to format messages like this (here could use [`console.rule`](https://rich.readthedocs.io/en/stable/console.html#rules))